src/curve.ts

```diff
@@ -44,7 +44,7 @@ export class Curve extends Element {
   //    cps: List of control points
   //    x_shift: pixels to shift
   //    y_shift: pixels to shift
-  constructor(from: Note, to: Note, options?: CurveOptions) {
+  constructor(from: Note, to: Note, options: Partial<CurveOptions>) {
```

Comment:
`Partial` is interesting here.

Reply:
I found this feature when investigating how to improve #1019 it is much better than double definitions.
It would make sense to merge #1019 in advance to this one to resolve the last `any`